pkg/apis/serving/v1alpha2/explainer_alibi.go

```diff
@@ -16,6 +17,11 @@ func (s *AlibiExplainerSpec) GetStorageUri() string {
 	return s.StorageURI
 }
 
+func (s *AlibiExplainerSpec) GetResourceRequirements() *v1.ResourceRequirements {
```

Comment:
Should this still be around?

I thought we were going to use GetContainer().Resources

Reply:
The reason why I did not use GetContainer().resource in predictor is as below:
>> Tried but the GetContainer function require modelName parameter which only available in metadata. If we are going to use that we need change validateEndpoint in inferenceservice_validation.go to take metadata as parameter, I think it's not worth doing so keep the original design. Please take a look, thanks.

And for explainer I originally used GetContainer().resource but then changed it as @yuzisun suggested to keep consistent with predictor.